docs/api/portable-stories.md

```diff
@@ -0,0 +1,430 @@
+import { render, screen } from '@testing-library/react';
+import { composeStory } from '@storybook/react';
+
+import meta, { Primary } from './Button.stories';
+
+test('onclick handler is called', () => {
+  // Returns a component that already contain all decorators from story level, meta level and global level.
+  const PrimaryStory = composeStory(Primary, meta);
+
+  const onClickSpy = vi.fn();
+  render(<PrimaryStory onClick={onClickSpy} />);
+  const buttonElement = screen.getByRole('button');
+  buttonElement.click();
+  expect(onClickSpy).toHaveBeenCalled();
+});
+```
+
+### Type
+
+<!-- prettier-ignore-start -->
+```ts
+(
+  story: Story export,
+  componentAnnotations: Meta,
+  projectAnnotations?: ProjectAnnotations,
+  exportsName?: string
+) => ComposedStoryFn
+```
+<!-- prettier-ignore-end -->
+
+### Parameters
+
+#### `story`
+
+(**Required**)
+
+Type: `Story export`
+
+Specifies which story you want to compose.
+
+#### `componentAnnotations`
+
+(**Required**)
+
+Type: `Meta`
+
+The default export from the stories file containing the [`story`](#story).
+
+#### `projectAnnotations`
+
+Type: `ProjectAnnotation | ProjectAnnotation[]`
+
+This parameter is provided for convenience. You should likely use [`setProjectAnnotations`](#setprojectannotations) instead. Details about the `ProjectAnnotation` type can be found in that function's [`projectAnnotations`](#projectannotations-2) parameter.
+
+This parameter can be used to [override](#overriding-globals) the project annotations applied via `setProjectAnnotations`.
+
+#### `exportsName`
+
+Type: `string`
+
+You probably don't need this. Because `composeStory` accepts a single story, it does not have access to the name of that story's export in the file (like `composeStories` does). If you must ensure unique story names in your tests and you cannot use `composeStories`, you can pass the name of the story's export here.
+
+### Return
+
+Type: `ComposedStoryFn`
+
+A single [composed story](#return).
+
+## setProjectAnnotations
+
+This API should be called once and before the tests run, typically in your testing tool's setup file. This will make sure that whenever `composeStories` or `composeStory` are used, the project annotations are taken into account as well.
+
+```ts
+// setup-tests.ts
+import { setProjectAnnotations } from '@storybook/react';
+import * as addonAnnotations from 'my-addon/preview';
+import * as previewAnnotations from './.storybook/preview';
+
+setProjectAnnotations([previewAnnotations, addonAnnotations]);
+```
+
+### Type
+
+```ts
+(projectAnnotations: ProjectAnnotation | ProjectAnnotation[]) => void
+```
+
+### Parameters
+
+#### `projectAnnotations`
+
+(**Required**)
+
+Type: `ProjectAnnotation | ProjectAnnotation[]`
+
+A set of project [annotations](#story-annotations) (those defined in `.storybook/preview.js|ts`) or an array of sets of project annotations, which will be applied to all composed stories.
+
+## Story annotations
```

Comment:
Maybe this should be called annotations? Becuase story annotations could be confused with story-level annotations 🤔 

I feel like there should be a specific docs page about annotations, which mentions about story level, component level (meta), and project level (preview.js). This is an important knowledge in Storybook, not necessarily related to portable stories

Reply:
I changed the heading to just "Annotations".

I agree with you on the dedicated docs page, but that would require a bunch of work to all related docs pages, which is well out of scope of this effort.